webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/controllers/Application.scala

```diff
@@ -20,12 +20,12 @@ class Application @Inject()(tracingDataStore: TracingDataStore, redisClient: Tra
     log() {
       for {
         before <- Fox.successful(System.currentTimeMillis())
-        _ <- tracingDataStore.healthClient.checkHealth
+        _ <- tracingDataStore.healthClient.checkHealth()
         afterFossil = System.currentTimeMillis()
         _ <- redisClient.checkHealth
         afterRedis = System.currentTimeMillis()
         _ = logger.info(
-          s"Answering ok for Tracingstore health check, took ${afterRedis - before} ms (FossilDB ${afterFossil - before} ms, Redis ${afterRedis - afterFossil} ms).")
+          s"Answering ok for Tracingstore health check, took ${afterRedis - before} ms (FossilDB at ${tracingDataStore.healthClient.authority}: ${afterFossil - before} ms, Redis at ${redisClient.authority} ${afterRedis - afterFossil} ms).")
```

Comment:
```suggestion
          s"Answering ok for Tracingstore health check, took ${afterRedis - before} ms (FossilDB at ${tracingDataStore.healthClient.authority} ${afterFossil - before} ms, Redis at ${redisClient.authority} ${afterRedis - afterFossil} ms).")
```

The colon is kinda confusing in combination with the colon separator between the address and port :)